EIPS/eip-908.md

```diff
@@ -117,4 +117,5 @@ TODO
 TODO
 
 ## Copyright
-Copyright and related rights waived via [CC0](https://creativecommons.org/share-your-work/public-domain/cc0/).
+
+Copyright and related rights waived via [CC0](../LICENSE.md).
```

Comment:
```suggestion
Copyright and related rights waived via [CC0](../LICENSE.md).
```

Reply:
Assuming that this was a desync error. Please unresolve if I've done something stupid.